src/lib/routes/edge-api/index.ts

```diff
@@ -59,12 +63,15 @@ export default class EdgeController extends Controller {
             middleware: [
                 this.openApiService.validPath({
                     tags: ['Edge'],
+                    summary:
+                        'Accepts a list of tokens, returns tokens that are valid and which projects they can access',
+                    description:
+                        'Send a list of tokens, only returns the valid ones',
```

Comment:
I think the description should be longer than the summary, also a small suggestion:
```suggestion
                    summary: 'Filter a list of tokens returning only the valid ones',
                    description:
                        'Accepts a list of tokens, and returns those that are valid with the projects they can access',
```